bundles/org.openhab.voice.googletts/README.md

```diff
@@ -77,7 +77,9 @@ In case you would like to setup these settings via a text file, you can edit the
 org.openhab.voice:defaultTTS=googletts
 org.openhab.voice:defaultVoice=googletts:XXX
 ```
+
```

Comment:
I saw your reverted commit using \`\`\`perl above. I'm wondering if \`\`\`ini would be useful here?

Reply:
Looks very similar for these blocks. I prefer to first discuss this before I create a new (separate) PR. @Confectrician might be able to add something too.

Reply:
It seems used already:

```shell
$ grep -A 2 -R "\`\`\`ini"
bundles/org.openhab.automation.jrubyscripting/README.md:   ```ini
bundles/org.openhab.automation.jrubyscripting/README.md-   org.openhab.automation.jrubyscripting:gems=openhab-scripting=~>5.0
bundles/org.openhab.automation.jrubyscripting/README.md-   org.openhab.automation.jrubyscripting:require=openhab/dsl
--
bundles/org.openhab.automation.jrubyscripting/README.md:```ini
bundles/org.openhab.automation.jrubyscripting/README.md-org.openhab.automation.jrubyscripting:gems=openhab-scripting=~>5.0
bundles/org.openhab.automation.jrubyscripting/README.md-org.openhab.automation.jrubyscripting:require=openhab/dsl
--
bundles/org.openhab.binding.lirc/README.md:```ini
bundles/org.openhab.binding.lirc/README.md-[Unit]
bundles/org.openhab.binding.lirc/README.md-Description=Linux Infrared Remote Control
--
bundles/org.openhab.persistence.dynamodb/README.md:```ini
bundles/org.openhab.persistence.dynamodb/README.md-profilesConfigFile=/etc/openhab2/aws_creds
bundles/org.openhab.persistence.dynamodb/README.md-profile=fooprofile
--
bundles/org.openhab.persistence.dynamodb/README.md:````ini
bundles/org.openhab.persistence.dynamodb/README.md-[fooprofile]
bundles/org.openhab.persistence.dynamodb/README.md-aws_access_key_id=testAccessKey
```

and rendered nicely also:

- https://www.openhab.org/addons/bindings/lirc/#supported-things
- https://www.openhab.org/addons/automation/jrubyscripting/#using-files

Reply:
Cool! Will adapt

Reply:
Done